op-service/sources/types.go

```diff
@@ -61,6 +61,9 @@ type RPCHeader struct {
 	// ParentBeaconRoot was added by EIP-4788 and is ignored in legacy headers.
 	ParentBeaconRoot *common.Hash `json:"parentBeaconBlockRoot,omitempty"`
 
+	// RequestsHash was added by EIP-7685 and is ignored in legacy headers.
```

Comment:
Would you be down to break this into its own PR? it would close https://github.com/ethereum-optimism/optimism/issues/13981

Reply:
then it would be really easy to cherry pick the commit for other teams that will depend on it

Reply:
Sure, split the work out here https://github.com/ethereum-optimism/optimism/pull/14022